app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class SessionsController < Devise::SessionsController
+  def create
+    user = User.find_by_email(sign_in_params[:email])
+
+    if user && user.valid_password?(sign_in_params[:password])
+      @current_user = user
```

Comment:
O Devise usa cookies para lidar com a autenticação entre as requisições, mas como no nosso caso estamos fazendo uma API precisar usar uma estratégia com tokens em vez de cookies.

A ideia é que quando o client passar o e-mail e senha pra fazer login nós precisamos passar um token pra ele de volta caso o login esteja correto, daí nas próximas requisições, (por exemplo, pra cadastrar uma task) o cliente vai passar esse token e vamos verificar se ele é válido ou não.

Achei esse artigo aqui que pode ajudar: https://dev.to/dhintz89/devise-and-jwt-in-rails-2mlj.

Reply:
Novo PR: https://github.com/bruno-azzi/tasks/pull/2